XMLHttpRequest/resources/auth8/corsenabled-no-authorize.py

```diff
@@ -0,0 +1,20 @@
+import imp
+import os
+
+def main(request, response):
+    response.headers.set('Access-Control-Allow-Origin', request.headers.get("origin"));
+    response.headers.set('Access-Control-Allow-Credentials', 'true');
+    response.headers.set('Access-Control-Allow-Methods', 'GET');
+    response.headers.set('Access-Control-Allow-Headers', 'x-user, x-pass');
+    response.headers.set('Access-Control-Expose-Headers', 'x-challenge, xhr-user, ses-user');
```

Comment:
I want the files to be different (and in different directories) to make sure the cached state of one test isn't interfering with the results of the next test. This matters for all Authorization: tests. Of course under the hood one of them could just include the other or something like we already do with authentication.py, but that's another level of "redirection" when a new developer tries to understand what's happening.. There's not that much code duplicated and I don't think it will need maintenance, so I think this copy'n'paste approach is actually good here.